AngularLanguageService/AngularLanguageClient.cs

```diff
@@ -44,6 +44,8 @@ public class AngularLanguageClient : ILanguageClient, ILanguageClientCustomMessa
 
         public object CustomMessageTarget => null;
 
+        public bool ShowNotificationOnInitializeFailed => throw new NotImplementedException();
```

Comment:
`false` would probably be a better default value here.

Reply:
Oh I was sure I had changed this. Fixing it!